docs/remote/containers.md

```diff
@@ -85,6 +85,8 @@ The easiest way to get started is to try one of the sample development container
 
 ![Select a sample from the list](images/containers/select-a-sample.png)
 
+> Note: If you already have VS Code and Docker installed, then you may use [open in dev container](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/vscode-remote-try-python. You may learn more about this and how to add it to your repos in the [create a dev container guide](./create-dev-container.md#add-configuration-files-to-a-repository).
```

Comment:
Was this meant to be a link or an example? 
If a link, it is missing a closing ')'.
If an example, should this be an inline code block?

Reply:
Good catch, this was meant to be a link! Fixing now.